src-docs/src/routes.js

```diff
@@ -500,6 +502,7 @@ const allRoutes = navigation.reduce((accummulatedRoutes, section) => {
 }, []);
 
 export default {
+  history: createHashHistory(), // eslint-disable-line react-hooks/rules-of-hooks
```

Comment:
```suggestion
  history: createHashHistory(),
```

No longer using a hook